Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/MainActivity.java

```diff
@@ -86,6 +94,8 @@ protected void onCreate(Bundle savedInstanceState)
   @Override
   protected void onResume()
   {
+    ThemeHelper.setCorrectTheme(this);
```

Comment:
Why is this only needed in MainActivity? If the criteria are that the activity must support themes and must allow launching SettingsActivity, EmulationActivity and CheatsActivity would also need it.

Reply:
I was thinking that since the user has to return to MainActivity after changing their theme in settings, this would be the only place you would need to correct the activity's theme (This only applies to activities that are "Paused"). But now that I'm thinking about it, I should also account for when someone tries to change the theme while playing a game and returns to EmulationActivity. That *should* be the only other place I need to add that right?

Reply:
No, CheatsActivity can also launch the settings.

Reply:
I don't think you can access the interface settings from CheatsActivity

Reply:
Oh. Yeah, that's true now that I think about it.